### PR TITLE
AO-20351-Update-Documentation-no-action 

### PR DESCRIPTION
**Pull request by ronilan**
This Pull Request:
- Fixes typos and move documentation files around.
- Merges into master. Everything else gets rebased on top.

Branch stays open for further iterations. Feel free to participate using Github editing UI.

### CODE REVIEW

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
         working-directory: .github/utils
 
       # results of step should be visually examined in https://my.appoptics.com/
-      - name: Start Server & use Client againt Staging
+      - name: Start Server & use Client againt Production
```
Comment (prettycoder) [inline .github/workflows/verify.yml:122]: did you say "fix typos"?

againt -> against

File: docs/developing.md [hunk 2]
```diff
@@ -57,8 +57,8 @@ The setup script ensures a "clean" work place with each run by removing artifact
 
 ## Instrumenting Using Local Code
 
-The Agent dev environment mounts the code into `usr/src/work` and anlos mounts two siblings directories if exist: `bindings` and `instrumented`.
-The former should hold a clone of the bindings repo, the latter is meant to act as a "playground" to build and instrument real node apps (e.g. an app that uses `express` as the web server and `morgan` for logging). This setup allows linking local versions of bindings and agent for end-to-end development.
+The Agent dev environment mounts the code into `usr/src/work` and also mounts two siblings directories if they exist: `bindings` and `instrumented`.
+The former should hold a clone of the bindings repo, the latter is for the `node-instrumneted` repo clone. It is meant to act as a "playground" to build and instrument real node apps (e.g. an app that uses `express` as the web server and `morgan` for logging). This setup allows linking local versions of bindings and agent for end-to-end development.
```
Comment (prettycoder) [inline docs/developing.md:61]: node-instrumneted -> node-instrumented

**Review by prettycoder — COMMENTED** [review]
2 more typos

File: .github/workflows/verify.yml (hunk 1 above)
Comment (ronilan) [reply .github/workflows/verify.yml:122]: I'm blind...

**Review by prettycoder — APPROVED** [review]

**Pull request merged**